tests/test_clide.py

```diff
@@ -0,0 +1,113 @@
+import os
+import uuid
+import random
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.sql import text as sa_text
+from sqlalchemy.orm import sessionmaker
+from opencdms.models import clide
+
+DB_URL = 'postgresql+psycopg2://postgres:password@127.0.0.1:35432/postgres'
+
+
+db_engine = create_engine(DB_URL)
+
+station_status_data = dict(
+    id=1,
+    status="ACTIVE",
+    description="test station status 1"
+)
+
+timezone_data = dict(
+    id=1,
+    tm_zone="UTC",
+    utc_diff=0,
+    description="UTC timezone"
+)
+
+station_data = dict(
+    id=random.randint(1000, 2000),
+    station_no=uuid.uuid4().hex[:15],
+    status_id=station_status_data["id"],
+    time_zone=timezone_data["tm_zone"],
+    region='UK'
+)
+
+
+@pytest.fixture
+def db_session():
+    Session = sessionmaker(bind=db_engine)
+    session = Session()
+    yield session
+    session.close()
+
+
+def setup_module(module):
+    with db_engine.connect() as connection:
+        with connection.begin():
+            db_engine.execute(sa_text(f'''TRUNCATE TABLE {clide.Station.__tablename__} RESTART IDENTITY CASCADE''').execution_options(autocommit=True))
```

Comment:
to be honest, it was easiest way to truncate table and resetting ID. I thought resetting ID would be nice because if we don't do so, it keeps increasing as postgresql does not automatically reset ID if a table is truncated like mysql does.

Reply:
Please let me know if I should remove this and and go with `Station.query.delete()`.

Reply:
Let's keep `TRUNCATE TABLE`, I think it's sensible.  Could you add a comment above the first set of truncates:
`# Postgresql does not automatically reset ID if a table is truncated like mysql does`

(We'll then refactor this into a generalised interface later)